sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -65,13 +65,17 @@
   // No instance of this class.
   private Samplers() {}
 
-  private static boolean isSampled(Decision decision) {
-    return decision.equals(Decision.RECORD_AND_SAMPLED);
+  static boolean isRecording(Decision decision) {
+    return Decision.RECORD.equals(decision) || Decision.RECORD_AND_SAMPLED.equals(decision);
+  }
+
+  static boolean isSampled(Decision decision) {
+    return Decision.RECORD_AND_SAMPLED.equals(decision);
   }
 
   /**
    * Returns a {@link SamplingResult} with the given {@code attributes} and {@link
-   * SamplingResult#isSampled()} returning {@code isSampled}.
+   * SamplingResult#getDecision()} ()} returning {@code decision}.
```

Comment:
Also tend to find it more readable to use something like `the provided {@link Decision}`.

Reply:
Done.